raiden/utils/serialization.py

```diff
@@ -82,10 +82,8 @@ def serialize_merkletree_layers(data) -> typing.List[str]:
 def deserialize_merkletree_layers(data: typing.List[str]):
     elements = map_list(deserialize_bytes, data)
     if len(elements) == 0:
-        return [
-            [],           # the leaves are empty
-            [bytes(32)],  # the root is the constant 0
-        ]
+        from raiden.transfer.state import make_empty_merkle_tree
```

Comment:
here is one of the cycles

Reply:
So no strong opinion about this, but wouldn't this be better if `make_empty_merkle_tree` was placed in a standalone module? similar case to `raiden/transfer/queue_identifier.py`

Reply:
Yep, Yep. May I do in another PR though?

Reply:
Sure :smile: